Packs/CortexAttackSurfaceManagement/Integrations/CortexAttackSurfaceManagement/CortexAttackSurfaceManagement.yml

```diff
@@ -0,0 +1,329 @@
+category: Utilities
+commonfields:
+  id: Cortex Attack Surface Management
+  version: -1
+configuration:
+- additionalinfo: Should be the web UI with `api-` appended to front.
+  display: Server URL (e.g., https://api-xsiam.paloaltonetworks.com)
```

Comment:
OK, in that case I think it would be better to add this example to the additional info, and remove it from the display name. 
In addition, it might be worth adding an explanation about the tenants (as discussed offline) and a link to the reference here as well.

Reply:
Addressed in Commit `Bulk 10/20/22`

Reply:
I updated the description file and readme in accordance as well.